.github/workflows/dco-check.yml

```diff
@@ -0,0 +1,13 @@
+name: DCO
+
+on:
+  pull_request:
+    branches: [main]
+
+permissions:
+  contents: read
+
+jobs:
+  check:
+    name: Organization
+    uses: openslide/.github/.github/workflows/dco-check.yml@main
```

Comment:
any reason this is not just openslide/.github/workflows/dco-check.yml?

Oh wait that would be this file. I really have to figure out github actions better.

Reply:
I thought it'd be cleaner to have the individual repos all invoke workflows from a separate org-level metadata repo, so we don't have redundant copies everywhere and don't have the main OpenSlide repo doing double duty as a library of CI workflows.